app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -538,6 +539,8 @@ class GithubWebhookSubscription extends SubscriptionHandler {
           base: Config.defaultBranch(slug),
         );
         await gitHubClient.issues.createComment(slug, pr.number!, body);
+        // Not that this should cancel all existing running tasks.
+        await _scheduleIfMergeable(pullRequestEvent);
```

Comment:
Do we want to schedule for `isReleaseChannelBranch` as well?

Reply:
BTW: I think we should separate the scheduling from `_checkForTests/_validateRefs`. Similar to the case of `opened`/`reopened`, shall we add the `_scheduleIfMergeable` logic to `edited` when detecting a branch mismatch?

Reply:
Hahaha dang it! Yeah I can do that cleanup.